clients/source.go

```diff
@@ -146,16 +146,16 @@ func (c *SourceClient) newManagedClient(ctx context.Context, path string) error
 				break
 			}
 			if errors.Is(err, errLogLineToLong) {
-				c.logger.Err(err).Str("line", string(line)).Msg("skipping too long log line")
+				c.logger.Info().Str("line", string(line)).Msg("truncated source plugin log line")
```

Comment:
I wonder if the word `source plugin` is needed. our logger has context so it is contexted with the source plugin name and all other properties.

Reply:
This logging happens in the context of the CLI logger so `module=cli`.
It comes from https://github.com/cloudquery/cloudquery/blob/9f33362ebc9989ca0f6f74eac9e5700657a18116/cli/cmd/sync.go#L79

You might be referring to the plugin logger that is created here https://github.com/cloudquery/plugin-sdk/blob/448232c8789350c8fb071902d33a5c5f07d2b82c/plugins/source/plugin.go#L153 from the serve command